lib/charms/loki_k8s/v0/log_proxy.py

```diff
@@ -44,7 +47,7 @@ def __init__(self, *args):
            ...
            try:
                self._log_proxy = LogProxyConsumer(
-                   charm=self, log_files=LOG_FILES, container_name=PEER
+                   charm=self, log_files=LOG_FILES, container_name=PEER, syslog=True
```

Comment:
Seems like this needs an update.

Reply:
I promise I'll open a PR with this after merging